devsetup/scripts/edpm-deploy-instance.sh

```diff
@@ -57,12 +58,18 @@ openstack compute service list
 openstack network agent list
 
 # Create an instance
-openstack server show test || {
-    openstack server create --flavor m1.small --image cirros --nic net-id=private test --security-group basic --wait
-    fip=$(openstack floating ip create public -f value -c floating_ip_address)
-    openstack server add floating ip test $fip
-}
-openstack server list
+for (( i=0; i<${NUMBER_OF_INSTANCES}; i++ )); do
+    NAME=test_${i}
+    openstack server show ${NAME} || {
+        openstack server create --flavor m1.small --image cirros --nic net-id=private ${NAME} --security-group basic --wait
+        fip=$(openstack floating ip create public -f value -c floating_ip_address)
+        openstack server add floating ip ${NAME} $fip
+    }
+    openstack server list --long
+
+    # check connectivity via FIP
+    ping -c4 $fip
+done
 
 # check connectivity via FIP
 ping -c4 $fip
```

Comment:
Is this needed anymore?

Reply:
gnahh, same as the other ... how could I miss. removed now